tests/benchmarks/micro_benchmarks/test_disk_performance.py

```diff
@@ -0,0 +1,493 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for disk-performance benchmark."""
+
+from pathlib import Path
+import os
+import unittest
+
+from superbench.benchmarks import BenchmarkRegistry, BenchmarkType, ReturnCode, Platform
+
+
+class DiskPerformanceTest(unittest.TestCase):
+    """Test class for disk-performance benchmark."""
+    def setUp(self):
+        """Method called to prepare the test fixture."""
+        # Create fake binary file just for testing.
+        os.environ['SB_MICRO_PATH'] = '/tmp/superbench/'
+        binary_path = os.path.join(os.getenv('SB_MICRO_PATH'), 'bin')
+        Path(binary_path).mkdir(parents=True, exist_ok=True)
+        self.__binary_file = Path(binary_path, 'fio')
```

Comment:
use `Path` directly? it's duplicate with `os.path`
```suggestion
        binary_path = Path(os.getenv('SB_MICRO_PATH'), 'bin')
        binary_path.mkdir(parents=True, exist_ok=True)
        self.__binary_file = binary_path / 'fio'
```

Reply:
Fixed